skbio/stats/composition.py

```diff
@@ -1105,6 +1238,75 @@ def _gram_schmidt_basis(n):
     return basis.T
 
 
+def sbp_basis(sbp):
+    """
+    Builds an orthogonal basis from a sequential binary partition (SBP). As
+    explained in [1]_, the SBP is a hierarchical collection of binary
+    divisions of compositional parts. The child groups are divided again until
+    all groups contain a single part. The SBP can be encoded in a
+    :math:`(D - 1) \times D` matrix where, for each row, parts can be grouped by
+    -1 and +1 tags, and 0 for excluded parts. The `sbp_basis` method was
+    originally derived from function `gsi.buildilrBase()` found in the R package
+    `compositions` [2]_. The ith balance is computed as follows
+
+    .. math::
+        b_i = sqrt(r_i*s_i / r_i+s_i) * log(g(x_r_i) / g(x_s_i))
```

Comment:
This isn't rendering correctly as is.  This will need to be changed to

```
    .. math::
        b_i = \sqrt{ \frac{r_i s_i}{r_i+s_i} }
        \ln \left( \frac{g(x_{r_i})}{g(x_{s_i})} \right)
```